shoop/apps/settings.py

```diff
@@ -105,3 +105,13 @@ def validate_templates_configuration():
         "The `django_jinja` template backend was not encountered in your TEMPLATES configuration. "
         "See the Shoop or Django-Jinja documentation on more information how to configure things correctly."
     )
+
+
+def reload_apps():
```

Comment:
Public API for this function should be in shoop.apps, not in shoop.apps.settings.